src/contracts/webauthn_account_contract/src/utils.nr

```diff
@@ -35,4 +66,12 @@ fn test_convert_array_to_field() {
     assert_eq(16711850, convert_to_array_to_field([0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,255,0,170]));
     assert_eq(max_field_value - 1, convert_to_array_to_field([48, 100, 78, 114, 225, 49, 160, 41, 184, 80, 69, 182, 129, 129, 88, 93, 40, 51, 232, 72, 121, 185, 112, 145, 67, 225, 245, 147, 240, 0, 0, 0]));
     assert_eq(max_field_value, convert_to_array_to_field([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ]));
-}
\ No newline at end of file
+}
+
+#[test]
+fn test_f() {
+    let input = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
+    assert_eq(std::hash::sha256([1]), prefix_sha256_hash(1, input));
+    assert_eq(std::hash::sha256([1, 2]), prefix_sha256_hash(2, input));
+    assert_eq(std::hash::sha256([1, 2, 3]), prefix_sha256_hash(3, input));
```

Comment:
Missing tests for
- [ ] 114
- [ ] 134 
- [ ] 243
- [ ] Length is not supported

Reply:
Tests have been added.